src/frontend/src/tests/lib/services/auth.services.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { signOut } from '$lib/services/auth.services';
+import { authStore } from '$lib/stores/auth.store';
+import { vi } from 'vitest';
+
+const rootLocation = 'https://oisy.com/';
+const activityLocation = 'https://oisy.com/activity';
+
+const mockLocation = (url: string) => {
+	Object.defineProperty(window, 'location', {
+		writable: true,
+		value: {
+			href: url,
+			reload: vi.fn()
+		}
+	});
+};
+
+describe('auth.services', () => {
```

Comment:
Looks good. Once this PR is merged, would you be interested adding more tests for the other feature of the service to reduce the test debt?

Reply:
Super, thx!